GraphView/graphview.py

```diff
@@ -1023,6 +1023,59 @@ def background_menu(self, event):
         self.menu = Gtk.Menu()
         self.menu.set_reserve_toggle_size(False)
 
+        menu_item = Gtk.CheckMenuItem(_('Show images'))
```

Comment:
I thought we had decided to move away from putting configuration settings in the context menus